e2ev2/tests/user-create.test.js

```diff
@@ -0,0 +1,94 @@
+const config = require('../src/config')
+const tu = require('../src/test-utils')
+const u = require('../src/utils')
+
+const browserTest = tu.browserTest(__filename)
+
+// Puppeteer's Page.click is not reliable. This is worth investigation.
+const jsClick = (page, sel) => page.waitForSelector(sel).then(eh => eh.evaluate(e => e.click()))
+
+const pressKey = async (page, keyName, count = 1) => {
+  for (let i = 0; i < count; ++i) {
+    await page.keyboard.press(keyName)
+  }
+}
+
+browserTest('create user', async browser => {
+  const page = browser.initialPage
+  const iframeSel = '[aria-label="terms of use and privacy statement"]>iframe'
+  const ackCheckboxSel = '[data-test-id="agreement-check"]'
+  const isAckCheckboxDisabled =
+    () => page.waitForSelector(ackCheckboxSel).then(eh => eh.evaluate(e => e.disabled))
+
+  await tu.useApiProxy(page)
+  await page.goto(config.urlRoot(), {waitUntil: 'networkidle0'})
+  await jsClick(page, '[data-test-id="login"]>div+div>[role="button"]') // Create Account
+
+  await page.waitForSelector(iframeSel) // Agreement Page
+  await page.evaluate(() => window.forceHasReadEntireAgreement())
+  await expect(isAckCheckboxDisabled()).resolves.toBe(false)
+  await jsClick(page, ackCheckboxSel)
+  await page.click('[role="button"]') // Next button
+
+  await page.waitForSelector('.p-dropdown') // Institution Page
+  await page.click('.p-dropdown')
+  await page.waitForSelector('[role="option"][aria-label="Broad Institute"]')
+  await page.click('[role="option"][aria-label="Broad Institute"]')
+  // Email needs to be real or it will bounce and account creation will fail.
```

Comment:
well, matching `@broadinstitute.org` anyway

Reply:
I tried `fake@broadinstitute.org` and could not create an account. The logs showed email bounces.